interactive_ai/tests/e2e/Makefile

```diff
@@ -49,11 +49,11 @@ tests: venv
 		echo "Selected tags: $(BDD_TAG)"; \
 		TAG_OPTION='--tags=$(BDD_TAG)'; \
 	fi; \
-	uv run -m behave $$TAG_OPTION $$TARGET --junit --junit-directory $(MAKEFILE_DIR)/reports/junit
+		export no_proxy=localhost,127.0.0.1,10.0.0.1/8 && uv run -m behave $$TAG_OPTION $$TARGET --no-skipped --junit --junit-directory $(MAKEFILE_DIR)/reports/junit
 
 test_wip: venv
 	@echo "Running WIP tests..."
-	uv run behave --wip features
+	export no_proxy=localhost,127.0.0.1,10.0.0.1/8 && uv run behave --wip features
```

Comment:
Were these changes accidentally included in the commit? I don't think we should hardcode proxy settings here

Reply:
I have added these on purpose, as the step to download files (e.g., zip of a trained project) does not work for most Intel developers as they will have `intel.com` in their no_proxy env var. We can remove it if you think it does not make sense to put it in.

Reply:
done